src/Compilers/CSharp/Test/Emit3/FieldKeywordTests.cs

```diff
@@ -1031,6 +1028,1438 @@ static void ReportField(FieldInfo field)
+                //     public override object P2 { get; set; }
+                Diagnostic(ErrorCode.ERR_NoSetToOverride, setter).WithArguments($"B1.P2.{setter}", "A.P2").WithLocation(4, 38),
+                // (5,32): error CS8051: Auto-implemented properties must have get accessors.
+                //     public virtual object P3 { set; }
+                Diagnostic(ErrorCode.ERR_AutoPropertyMustHaveGetAccessor, setter).WithLocation(5, 32),
+                // (5,33): error CS0545: 'B1.P3.get': cannot override because 'A.P3' does not have an overridable get accessor
+                //     public override object P3 { get; set; }
+                Diagnostic(ErrorCode.ERR_NoGetToOverride, "get").WithArguments("B1.P3.get", "A.P3").WithLocation(5, 33));
+
+            string sourceB2 = $$"""
+                class B2 : A
+                {
+                    public override object P1 { get => field; {{setter}} { } }
+                    public override object P2 { get => field; {{setter}} { } }
+                    public override object P3 { get => field; {{setter}} { } }
+                }
+                """;
+            comp = CreateCompilation([sourceA, sourceB2], targetFramework: TargetFramework.Net80);
+            comp.VerifyEmitDiagnostics(
+                // (4,47): error CS0546: 'B2.P2.set': cannot override because 'A.P2' does not have an overridable set accessor
+                //     public override object P2 { get => field; set { } }
+                Diagnostic(ErrorCode.ERR_NoSetToOverride, setter).WithArguments($"B2.P2.{setter}", "A.P2").WithLocation(4, 47),
+                // (5,32): error CS8051: Auto-implemented properties must have get accessors.
+                //     public virtual object P3 { set; }
+                Diagnostic(ErrorCode.ERR_AutoPropertyMustHaveGetAccessor, setter).WithLocation(5, 32),
+                // (5,33): error CS0545: 'B2.P3.get': cannot override because 'A.P3' does not have an overridable get accessor
+                //     public override object P3 { get => field; set { } }
+                Diagnostic(ErrorCode.ERR_NoGetToOverride, "get").WithArguments("B2.P3.get", "A.P3").WithLocation(5, 33));
+
+            string sourceB3 = $$"""
+                class B3 : A
+                {
+                    public override object P1 { get => field; }
+                    public override object P2 { get => field; }
+                    public override object P3 { get => field; }
+                }
+                """;
+            comp = CreateCompilation([sourceA, sourceB3], targetFramework: TargetFramework.Net80);
+            comp.VerifyEmitDiagnostics(
+                // (3,28): error CS8080: Auto-implemented properties must override all accessors of the overridden property.
+                //     public override object P1 { get => field; }
+                Diagnostic(ErrorCode.ERR_AutoPropertyMustOverrideSet, "P1").WithLocation(3, 28),
```

Comment:
I always forget about this. Yes I feel fine with saying that if you use the field keyword then you must override all the accessors. It feels like if you forgot to override one and you are using that keyword then something is going wrong, a value is being dropped on the floor somewhere.